CHANGES.rst

```diff
@@ -1,27 +1,27 @@
 Changes
 =======
 
-Version 0.8.0 (UNRELEASED)
+Version 0.8.0 (2021-11-22)
 --------------------------
 
-- Adds new command ``quota-show``.
-- Adds possibility to filter by workflow status and search by workflow name to ``list`` command.
-- Adds job start and finish times to ``logs`` command output.
-- Adds possibility to filter by file name, size and last modified values to ``ls`` command.
-- Adds new command ``info`` that retrieves general information about the cluster.
+- Adds new command ``quota-show`` to retrieve information about total CPU and Disk usage and quota limits.
+- Adds new command ``info`` that retrieves general information about the cluster, including the list all the available workspace paths.
 - Adds new ``--server-capabilities`` flag to ``validate`` command.
-- Adds the possibility to filter and display all the runs of a given workflow to the ``list`` command.
-- Changes ``delete`` to prevent workflow hard deletion.
-- Changes ``list`` to display interactive session status.
+- Adds support for ``outputs.directories`` in ``reana.yaml``.
+- Adds support for Snakemake workflow submission and validation.
+- Changes ``list`` command to add the possibility to filter by workflow status and search by workflow name via ``--filter`` option.
+- Changes ``list`` command to add the possibility to filter and display all the runs of a given workflow via ``-w`` option.
 - Changes ``list`` command to stop including workflow progress and workspace size by default. Adds
+- Changes ``list`` to display interactive session status.
+- Changes ``logs`` command to add job start and finish times to the output.
   ``--include-progress`` and ``--include-workspace-size`` options to add them to the output.
+- Changes ``ls`` command to add the possibility to filter by file name, size and last modified values via ``--filter`` option.
+- Changes ``du`` command to add the possibility filter by file name and size via ``--filter`` option.
+- Changes ``delete`` to prevent workflow hard deletion.
 - Changes Yadage workflow specification loading to be done in ``reana-commons``.
+- Changes ``cwltool`` version to ``3.1.20210628163208``.
 - Removes support for Python 2.
```

Comment:
- Removes support for Python 2.7. Please use Python 3.6 or higher from now on.

Reply:
> Please use Python 3.6 or higher from now on.

Can you please implement this in `setup.py` with `python_requires`? If support for Python 2 is dropped without a `python_requires>=3.6` in `setup.py` then if a Python 2 user attempts to install `reana-client` from PyPI they won't get the last Python 2 supported release but rather a broken (for Python 2) release.

I guess I could have made this PR in the time it has taken me to write this, so I'll do so now.